README.md

```diff
@@ -149,21 +149,26 @@ Results table for the EDD.
 **Uses:**
 
 - Process data into one of the tables in the format accepted by EQuIS
-- Censor values less than or equal to the MDL
-- Raise the J-R flag for observations greater than the MDL but less than
-  or equal to the LQL
+- Censor values less than the MDL and LQL and assign
+  Result_Detection_Condition approriately
 
 **Limitations:**
 
 - Users still need to create Activities table for every deliverable and
   the Projects and Locations tables when they require edits.
 - Within the Results table, users still need to define Activity_ID,
   modify Result_Status (set to Pre-Cert since additional QC is needed),
-  add flags, and conduct the rest of their own QC processes
+  add flags to Result_Qualifier, and conduct the rest of their own QC
+  processes
 
-Here we demonstrate the useage of this function. Read the data, create
-the Results table of the EDD, and work with it in R without writing the
-data to any files:
+For further information regarding the EQuIS EDD, please refer to the
+[NPS EQuIS Resources
+website](https://doimspp.sharepoint.com/sites/nps-nrss-wrdiv/SitePages/DMEQuIS.aspx?xsdata=MDV8MDJ8fDE1NTcyM2EwNmEyODQ3NThhZDM5MDhkYzUzNDVjZjUwfDA2OTNiNWJhNGIxODRkN2I5MzQxZjMyZjQwMGE1NDk0fDB8MHw2Mzg0NzY4MDY0NzU3ODc1Nzd8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPbTFsWlhScGJtZGZUbXBuTVU5RWF6UlBWR2QwV21wck5FMXBNREJPZWtreVRGUm9iRTFxWjNSYVZFWnBUMFJWTTAxNmJHaFBSRUpxUUhSb2NtVmhaQzUyTWk5dFpYTnpZV2RsY3k4eE56RXlNRGd6T0RRMk1qVXd8MDU3NTdhODQyYWI2NDI5MGFkMzkwOGRjNTM0NWNmNTB8Yjg5ZDU1YmI3ZjVhNDNhNmJlODQwMzU5NDgyNTM1ZmM%3D&sdata=TjVsUVVHQS84VDQ2NUlRZmNzMlE3R2hRRWI0MDdlaVRIL0hkNnAxMmRSQT0%3D&ovuser=0693b5ba-4b18-4d7b-9341-f32f400a5494%2Cavolk%40nps.gov&OR=Teams-HL&CT=1719255259520&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiI0OS8yNDA1MTYyMjIyMyIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D),
```

Comment:
I'm assuming this is an internal-only site? If so, I'd add some kind of note to indicate that, so that folks outside the network won't think it's a broken link